content/docs/guides/components/modeldb.md

```diff
@@ -0,0 +1,98 @@
++++
+title = "ModelDB"
+description ="ModelDB - A system to manage machine learning models"
+weight = 10
+toc = true
+[menu]
+[menu.docs]
+  parent = "components"
+  weight = 5
++++
+
+## Using ModelDB
+
+After ModelDB is deployed using kfctl.sh and the modeldb-db, modeldb-frontend and modeldb-backend pods are running - 
+
+1)  Port forward the modeldb-backend pod to port 6543 - 
+
+```
+kubectl get pods -n kubeflow # Find your modeldb-backend pod
+kubectl port-forward [modeldb-backend pod] 6543:6543 -n kubeflow
+``` 
+2)  Create a ModelDB syncer
```

Comment:
Can you make this a new section with a descriptive heading e.g. "Loading metrics from Python"

Reply:
To use the ModelDB lib on the client machine, they'll first have to do a "pip install modeldb" to get the lib

Reply:
I'd recommend we say something like: "Open up a Jupyter notebook you will use for model training. Make sure that the modeldb libraries have been installed already or install via 'pip install modeldb'."

Reply:
ok